modules/http-impl/src/main/java/org/mule/services/http/impl/service/server/grizzly/GrizzlyServerManager.java

```diff
@@ -104,10 +98,7 @@ public GrizzlyServerManager(String threadNamePrefix, HttpListenerRegistry httpLi
     // Set filterchain as a Transport Processor
     transport.setProcessor(serverFilterChainBuilder.build());
 
-    idleTimeoutExecutorService =
```

Comment:
Yes, this was causing a memory leak since having an executor per container caused it's queue to grow without end. I've introduced the IdleExecutor per server address so that they can be destroyed once the server goes down.